lib/kits/core-ui/components/helpers/component.ts

```diff
@@ -426,7 +426,9 @@ export const closeSidebar = () => {
 export const displayOfferMessage = (viewId, message) => {
     if (!message) return;
 
-    const offerContainer = document.querySelector(`[data-id="${viewId}"]`);
+    const offerContainer = document.querySelector(
```

Comment:
Don't understand this. Why any 3.0 specific changes?

Reply:
Wouldn't you instead do clickedEl.closest('[data-role=offer][data-id]')? This is more stable.

Reply:
Yes! Thanks for the idea! 😄 
Here 🙂 : https://github.com/tjek/tjek-js-sdk/pull/274/commits/ba46a73058c678c3d04e8ca11edf0e54436b5a81